addons/hide-monitors/userstyle.css

```diff
@@ -0,0 +1,3 @@
+.sa-hide-monitors {
+  opacity: 0;
```

Comment:
I wonder if they should be just barely visible so you can tell where they are if you're trying to click and it's not working.

Reply:
Maybe it could be a three state toggle? `Monitors shown` -> `Monitors barely shown` -> `Monitors hidden`?

Reply:
Whether they were hidden or barely visible could also be set in the addon's settings.